timeseries/src/main/java/com/hurence/logisland/timeseries/sax/SaxAnalyzer.java

```diff
@@ -0,0 +1,75 @@
+package com.hurence.logisland.timeseries.sax;
+
+import com.sun.javafx.collections.MappingChange;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+
+public class SaxAnalyzer {
+
+    public static Map<Integer, Integer> countOccurrence(int[] numbersToProcess) {
+        int[] possibleNumbers = new int[30];
+        Map<Integer, Integer> result = new HashMap<Integer, Integer>();
+
+        for (int i = 0; i < numbersToProcess.length; ++i) {
+            possibleNumbers[numbersToProcess[i]] = possibleNumbers[numbersToProcess[i]] + 1;
+            result.put(numbersToProcess[i], possibleNumbers[numbersToProcess[i]]);
+        }
+
+        return result;
+    }
+    public static List<Integer> saxThreshold(String str){
```

Comment:
Would be nice to add an comment on the function using standard notations in javadoc. You should at least explain what the algorithm is... and give may be an example or a reference to a known algo ;)